src/pages/project/project_wizard/NRFProjectOption.tsx

```diff
@@ -0,0 +1,21 @@
+import { Box, Typography } from '@mui/material';
+import { useTranslation } from 'react-i18next';
+import { NfrProject } from '../../../types/project.types';
+import { getPeriodString } from '../../../utils/general-helpers';
+import { getLanguageString } from '../../../utils/translation-helpers';
+
+interface NFRProjectOptionProps {
+  project: NfrProject;
+}
+
+export const NFRProjectOption = ({ project }: NFRProjectOptionProps) => {
```

Comment:
samme casing som ellers? gjelder filnavnet også
```suggestion
interface NfrProjectOptionProps {
  project: NfrProject;
}

export const NfrProjectOption = ({ project }: NfrProjectOptionProps) => {
```